setup.md

```diff
@@ -340,16 +340,22 @@ new lesson is `data-cleanup`.
     ~~~
     {: .language-bash}
 
-	This will ensure that you are using the most "stable" version of the
-	template repository. Since it's being actively maintained by the
-	Software Carpentry community, you could end up using a development branch
-	that contains experimental (and potentially not working) features without
-	necessarily realising it. Switching to the `gh-branch` ensures you are
-	using the "stable" version of the template.
+    This will ensure that you are using the most "stable" version of the
+    template repository. Since it's being actively maintained by The
+    Carpentries community, you could end up using a development branch
+    that contains experimental (and potentially not working) features without
+    necessarily realising it. Switching to the `gh-pages` branch ensures you are
+    using the "stable" version of the template.
+
+11. Create all of the boilerplate files that cannot be put into the styles repository
+    (because they would trigger repeated merge conflicts), by running:
+    ~~~
+    $ python bin/lesson_initialize.py
+    ~~~
+    {: .language-bash}
 
-11. Run `bin/lesson_initialize.py` to create all of the boilerplate files
-    that cannot be put into the styles repository
-    (because they would trigger repeated merge conflicts).
+    *QUESTION:* Do we need to add, commit, push these files? Or edit them all first?
```

Comment:
I think here it would be good to have people edit the files first, and maybe suggest running `make lesson-check`.

Reply:
Ok, I will try to add those instructions.

Reply:
Isn't `make lesson check`, etc., covered in `07-checking.md`? In that case we don't need the instructions here, we can remove the QUESTION.

Reply:
:+1: